hasjob/templates/macros.html

```diff
@@ -66,8 +66,8 @@ <h1 itemprop="name">{{ headline or post.headline }}</h1>
         &middot; {% if kiosk %}{{ post.company_url|cleanurl }}{% else %}<a href="{{ post.company_url }}" target="_blank" rel="nofollow">{{ post.company_url|cleanurl }}</a>{% endif %}
       {%- endif %}
       {% if not post.is_announcement() %}
-        &middot; <a href="{{ url_for('browse_by_type', name=post.type.name) }}">{{ post.type.title }}</a>
-        &middot; <a href="{{ url_for('browse_by_category', name=post.category.name) }}">{{ post.category.title }}</a>
+        &middot; <a href="{{url_for('index', t=post.type.name)}}">{{ post.type.title }}</a>
+        &middot; <a href="{{url_for('index', c=post.category.name)}}">{{ post.category.title }}</a>
```

Comment:
Good time to add `url_for` methods to JobType and JobCategory (or maybe after hasgeek/coaster#77 is done).